daemon/bitcoin/mempool/mempool_space.go

```diff
@@ -0,0 +1,168 @@
+package mempool
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/40acres/40swap/daemon/bitcoin"
+	"github.com/btcsuite/btcd/wire"
+)
+
+const BaseURL = "https://mempool.elenpay.tech/api"
```

Comment:
this is OS Idk if we want to say ep here

Reply:
you're right, will remove